router/handlers.go

```diff
@@ -1,20 +1,58 @@
 package router
 
 import (
+	"fmt"
 	"html/template"
 	"log"
 	"net/http"
+	"strings"
 )
 
 var (
 	AppTitle = "Budget Helper"
 )
 
 var (
-	tmplHome  = template.Must(template.ParseFiles("./templates/pages/home.html", "./templates/base.html"))
-	tmplLogin = template.Must(template.ParseFiles("./templates/pages/login.html", "./templates/base.html"))
+	tmplHome      = addTemplate("pages/home.html")
+	tmplError     = addTemplate("pages/error.html")
+	tmplLogin     = addTemplate("pages/login.html")
+	tmplRegister  = addTemplate("pages/register.html")
+	tmplDashboard = addTemplate("pages/dashboard.html")
+	GET           = "GET"
```

Comment:
Move these to `const` scope instead of var; we should signal that they will never change.